CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ The changes are relative to the previous release, unless the baseline is specifi
   avifCropRectFromCleanApertureBox() and avifCleanApertureBoxFromCropRect().
 * Ignore descriptive properties associated after transformative ones.
 * Reject non-essential transformative properties.
+* Treat avifenc --stdin as a regular positional file path argument.
```

Comment:
I haven't read the pull request. I have a question: where is the position of this fictitious file path argument?

Here is my expectation:

- If the output file path is specified without using the `-o` option, the fictitious input file path argument is right before the output file path argument.
- If the `-o` option is used, the fictitious input file path argument is at the end of the command line.

Is this what you implemented?

Reply:
It makes no sense to use `:u` with `--stdin` because no other input is allowed. So how `:u` behaves with `--stdin` does not matter, as long as nothing is silently ignored.

I find it simpler to understand and to implement if `--stdin` can just be replaced with `/path/to/in.y4m` with identical behavior. I do not think it has to be mentioned in `--help` because there is no other input file name allowed, so it does not really matter where `--stdin` is.

Happy to revisit if needed though.

Reply:
I agree that it's easier to understand if `--stdin` can be replaced with `/path/to/in.y4m` with identical behavior. This simple rule is only mentioned in the changelog. It would be good to document this in the help message.

Also, this rule needs to make an exception for the `avifenc out.avif --stdin` case. The exception might be worth documenting.

Reply:
> It would be good to document this in the help message.

I disagree. It does not provide any benefit as long as "No other input is allowed": since there is a single input and a single output, the order of the other arguments should not matter. The same behavior as if replaced by a file path is somewhat expected, and if it is not the case, it does not really matter either because there is a single input and a single output.  
So describing that in the help message would just bring confusion in my opinion.